tests/snippets/exceptions.py

```diff
@@ -1,29 +1,37 @@
+def exceptions_eq(e1, e2):
+    return type(e1) is type(e2) and e1.args == e2.args
+
+def round_trip_repr(e):
+    assert exceptions_eq(e, eval(repr(e)))
```

Comment:
Nitpicky, but maybe this should return and the assert should be where it's called? That way an error would show the line with the exception variable instead of just the parameters.

Reply:
I based it on [the `round_trip_test()` from `json_snippet.py`](https://github.com/RustPython/RustPython/blob/master/tests/snippets/json_snippet.py#L4-L8) where it has similar logic. But I agree it’d be a bit nicer to move asserts out of the function. Shall I change it just here, or in both snippets (I haven’t found any other round-tripy utils).